wgpu-native/Cargo.toml

```diff
@@ -40,9 +35,10 @@ gfx-backend-dx12 = { version = "0.3.0", optional = true }
 gfx-backend-dx11 = { version = "0.3.0", optional = true }
 gfx-backend-metal = { version = "0.3.0", optional = true }
 gfx-backend-gl = { version = "0.3.0", optional = true }
-parking_lot = { version = "0.9" }
-rendy-memory = { version = "0.4" }
-rendy-descriptor = { version = "0.4" }
+parking_lot = "0.9"
+raw-window-handle = "0.1"
```

Comment:
Technically I guess somebody could prefer for this to be optional too, but it's probably fine to always require it